ixnetwork/api/ixn_rest.py

```diff
@@ -75,28 +75,35 @@ def disconnect(self):
     #
 
     def waitForComplete(self, request, response, timeout=128):
```

Comment:
While we visit the code, could you please:
- rename wait_for_complete
- add type hints
- add one liner doc string """ Wait for non-blocking REST call to complete """ or something similar?

Reply:
Ok -- I'll do this and give it some py3💝 etc. later today.

Regarding the previous comment, please also let me know what you'd like to do with checking the `state` -- should we drop the final '`elif`' block below?  I don't have an environment in place to test against different API versions so don't know if it would be good to do or not.

```python
            if state == 'error':
                result = json.loads(response.content)['result'].strip()
                raise TgnError(f'wait for post {response.url} failed - {result}')
            elif state == 'success':
                return response
            elif state not in ['unknown', 'in_progress', 'down']:
                return response
```

Thanks for reviewing.